source/code-generation/query-converter/atlas-log-in.txt

```diff
@@ -0,0 +1,76 @@
+.. _rm-atlas-log-in:
+
+=================
+Log In with Atlas
+=================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can sign in to your Atlas account inside Relational Migrator. When 
+you log in with your Atlas account in Relational Migrator, you can 
+access the :ref:`Query Converter <rm-query-converter>` feature.
+
+Before you Begin
+----------------
+
+To log in with Atlas, you must have an existing :atlas:`Atlas account 
+</tutorial/create-atlas-account>` and a 
+:atlas:`deployment </create-connect-deployments>`. 
+
+About this Task
+---------------
+
+- To log in with your Atlas account, Relational Migrator must be running 
+  on localhost on one of the following ports: ``8278``, ``8080``, or 
+  ``443``. If you run Relational Migrator on ports not listed above, 
+  the sign-in process fails.
```

Comment:
**Question**

For tech review, not sure if we need additional content instructing users how they can change this port in the app? This may differ if your using docker vs binary, but currently we dont have surface area on the config / settings page.

```suggestion
- To log in with your Atlas account, Relational Migrator must be running 
  on localhost on one of the following ports: ``8278``, ``8080``, or 
  ``443``. If you run Relational Migrator on ports not listed above, 
  the sign-in process fails.
```